infrastructure/network/netadapter/server/grpcserver/protowire/messages.proto

```diff
@@ -47,9 +47,10 @@ message KaspadMessage {
     ReadyMessage ready = 50;
     BlockWithTrustedDataV4Message blockWithTrustedDataV4 = 51;
     TrustedDataMessage trustedData = 52;
-    RequestIBDChainBlockLocatorMessage requestIBDChainBlockLocator = 53;
-    IbdChainBlockLocatorMessage ibdChainBlockLocator = 54;
-    RequestAnticoneMessage requestAnticone = 55;
+    RequestNextPruningPointAndItsAnticoneBlocksMessage requestNextPruningPointAndItsAnticoneBlocks = 53;
```

Comment:
Please set this message's ID to 56 and return the rest to what they were. This is to avoid messages getting parsed incorrectly between versions.